paper.tex

```diff
@@ -355,6 +355,102 @@ \section*{Recent technical improvements}
 
 \subsection*{Random}
 
+The NumPy random module provides pseudorandom numbers from a wide range of
+distributions. In legacy versions of NumPy, simulated random values are produced
+by a RandomState object that: handles seeding and state initialization; wraps
+the core pseudorandom number generator based on a 32-bit implementation of
+MT19937; interfaces with the underlying code that transforms random bits into
+deviates from other distributions; and supplies a singleton instance exposed in
+the root of the random module.
+
+The RandomState object makes a compatibility guarantee so that a fixed seed and
+sequence of function calls produce the same set of values. This guarantee has
+slowed progress since improving the underlying code requires extending the API
+with additional keyword arguments. This guarantee continues to apply to
+RandomState. 
+
+NumPy 1.17 introduced a new API for generating random numbers that use a more
+flexible structure that can be extended by libraries or end-users. The new API
+is build using components separate the tasks needed when generating random
```

Comment:
"is built using separate components for each of the tasks ...

Reply:
Thanks.